install_config/provisioners.adoc

```diff
@@ -15,35 +15,37 @@ toc::[]
 
 An external provisioner is an application that enables dynamic provisioning for
 a particular storage provider. External provisioners can run alongside the
-provisioner plug-ins provided by {product-title} and are configured in a
-similar way using xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc[_StorageClasses_]. However, being external, they can be deployed
-and updated independently of {product-title} on an as-needed basis.
+provisioner plug-ins provided by {product-title} and are configured in a similar
+way as the *_StorageClass_* objects are configured, as described in the  xref:../install_config/persistent_storage/dynamically_provisioning_pvs.adoc#[Dynamic Provisioning and Creating Storage Classes] section.
+Since these provisioners are external, you can deploy and update them
+independently of {product-title}.
 
 [[provisioners-before-you-begin]]
 == Before You Begin
+This section provides information for preparing to use Ansible and
+includes information about adding parameters to the Ansible inventory file, to
+configure which external provisioners to install and their configuration. An link:https://github.com/openshift/openshift-ansible/blob/master/playbooks/common/openshift-cluster/openshift_provisioners.yml[Ansible Playbook] is also available to deploy and upgrade external provisioners.
```

Comment:
I'm not sure if I understand the above sentence. Maybe:

"This section provides information for preparing to use Ansible and includes information about adding parameters to the Ansible inventory file, which external provisioners to configure, and how to install them."

I'd also argue that the sentence isn't needed, because the reader is about to read those sections and should be able to tell from the upcoming titles.